### PR TITLE
User management bottom bar state saved

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/7351

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by JanAckermann** [comment]
Bad: save state stays if user changes

File: packages/web-app-user-management/tests/unit/views/Users.spec.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ describe('Users view', () => {
   })
 
   describe('method "editUserGroupAssignments"', () => {
-    it('should show message on success', async () => {
+    it('should emit event success', async () => {
```
Comment (JanAckermann) [inline packages/web-app-user-management/tests/unit/views/Users.spec.js:78]: on....

File: packages/web-app-user-management/src/views/Users.vue [hunk 2]
```diff
@@ -484,11 +484,8 @@ export default defineComponent({
          */
         this.selectedUsers = [editUser]
 
-        this.showMessage({
-          title: this.$gettext('User was edited successfully')
-        })
+        bus.publish('app.user-management.entity.saved')
       } catch (error) {
-        console.error(error)
```
Comment (kulmann) [inline packages/web-app-user-management/src/views/Users.vue:491]: Why did you delete this line? 🤔 IMO makes sense to log errors to the console.

Comment (JanAckermann) [reply packages/web-app-user-management/src/views/Users.vue:491]: ooops, recovered

Comment (kulmann) [reply packages/web-app-user-management/src/views/Users.vue:491]: Thanks :-)

**Review by kulmann — APPROVED** [review]
Good improvement 💯
